common_helper.go

```diff
@@ -186,7 +186,7 @@ func validateTenantFromDB(tenant tenant) (bool, dbError) {
 	db := getDB()
 	rows, err := db.Query("SELECT 1 FROM edgex_data_scope"+
 		" where org = ? and env = ?", tenant.Org, tenant.Env)
-
+	defer rows.Close()
```

Comment:
I showed this to Pooja couple weeks back. Thought she was going to take care of this.

Reply:
May be there was some confusion because I got the impression that Sundar was going to make this change as he had already tested it locally. Thanks for taking care of it.